drivers/sensor/nrf5/temp_nrf5.c

```diff
@@ -16,6 +16,9 @@ LOG_MODULE_REGISTER(TEMPNRF5);
 #include "nrf_common.h"
 
 
+static NRF_TEMP_Type *const nrf_temp_addr =
+		(NRF_TEMP_Type *)DT_NORDIC_NRF_TEMP_TEMP_BASE_ADDRESS;
```

Comment:
Agreed. I thought it was decided that this was the way to go based on the nrfx_uart.c driver implementation. I will revert to using the NRF_TEMP. Then this can be revisited whenever the issue you referenced is resolved.

Reply:
But i will be removing the volatile *temp pointer from the functions as i dont see the need to create a pointer to the struct that is used. Agree? So the temp_nrf5_sample_fetch() function will have "volatile NRF_TEMP_Type *temp = NRF_TEMP;" removed, and "temp->TASKS_START = 1;" will be "NRF_TEMP->TASKS_START = 1;" instead.

Reply:
> But i will be removing the volatile *temp pointer from the functions as i dont see the need to create a pointer to the struct that is used. Agree?

Yes. I have no idea why this pointer was introduced. I don't see a need for it, too. It only obscures the code IMHO.